### PR TITLE
Updated Details Bar to match latest designs

### LINKED ISSUE #1107: Details bar UX options
The way we're currently injecting the details bar into published projects isn't the most reliable. We're adjusting the body padding to make room for the bar, but that doesn't take into account projects that have absolutely positioned (or position: fixed) content. People could also adjust their own body padding in the project CSS, which would conflict with ours. Here's a proposal for a lighter version of details bar...

details-bar-proposal

This version is a small "button" that sits overtop of the content and expands when hovered or clicked. We don't really have to make changes to the published project to accomodate it, we just have to make sure it has a high enough z-index.

I'm not sure about the positioning though, and am worried that it de-emphasizes the brand & editor.

### PR DESCRIPTION
**Pull request by flukeout**
Implements and fixes #1107

### CODE REVIEW

**Comment by flukeout** [comment]
@humphd R?

File: public/resources/remix/index.js [hunk 1]
```diff
@@ -68,15 +80,21 @@
...
+    '    <span class="icon"></span>\n' +
+    '  </a>\n' +
+    '  <h1 class="remix-project-title"></h1>\n' +
+    '  <div class="remix-project-meta">\n' +
+    '    By <span class="remix-project-author"></span>\n' +
+    '  </div>\n' +
+    '  <div class="details-bar-remix-button-wrapper">\n' +
+    '    <a title="Remix this project with Thimble" class="details-bar-remix-button">Remix</a>\n' +
+    '  </div>\n' +
+    '  <div class="close-details-bar" href="#"><img src="' + metadata.host + '/resources/remix/close-x.svg" /></div>\n' +
```
Comment (humphd) [inline public/resources/remix/index.js]: Why `href="#"` on a div?

File: public/resources/remix/index.js [hunk 2]
```diff
@@ -68,15 +80,21 @@
...
-    '    <a class="thimble-logo" href="https://thimble.mozilla.org">\n' +
-    '        <span class="remix-icon"></span>\n' +
-    '    </a>\n' +
-    '    <h1 class="remix-project-title"></h1>\n' +
-    '    <div class="remix-project-meta"><a class="remix-project-author" href="#"></a></div>\n' +
-    '    <div class="details-bar-remix-button-wrapper">\n' +
-    '        <a class="details-bar-remix-button">Remix</a>\n' +
-    '    </div>\n' +
+    '<div class="details-bar collapsed cleanslate">\n' +
+    '  <a class="thimble-logo" title="Thimble by Mozilla" href="thimble.mozilla.org">\n' +
```
Comment (humphd) [inline public/resources/remix/index.js]: add `https://` to the front of your `href="thimble..."`

File: public/resources/remix/index.js [hunk 3]
```diff
@@ -6,20 +6,32 @@
...
-    $(window).on("scroll", function() {
-      var scrollDistance = $(this).scrollTop();
-      if(scrollDistance >= detailsBarHeight) {
-        detailsBar.addClass("scrolled");
-      } else {
-        detailsBar.removeClass("scrolled");
-      }
+  function setupBar(){
+    var isTouchDevice = 'ontouchstart' in document.documentElement;
+    var detailsBar = document.querySelector(".details-bar");
```
Comment (humphd) [inline public/resources/remix/index.js]: The rest of this file uses jquery, can you update this function to do the same, so we don't mix styles?

File: public/resources/remix/index.js [hunk 4]
```diff
@@ -120,11 +138,10 @@
     var $$ = $.noConflict(true);
     $$(document).ready(function($) {
       var metadata = getMetadata($);
-
       injectAnalytics($);
       injectStyleSheets($, metadata);
       injectDetailsBar($, metadata);
-      customizeScroll($);
+      setupBar();
```
Comment (humphd) [inline public/resources/remix/index.js]: Pass `$` here as an arg so you can use jquery in `setupBar`

File: public/resources/remix/style.less [hunk 5]
```diff
@@ -127,23 +79,182 @@
...
+    }
+
+    &.collapsed {
+      width: 112px !important;
+      padding: 11px 11px 11px 14px !important;
+      box-shadow: 0 1px 1px rgba(0,0,0,.2) !important;
+      box-shadow: none !important;
+      text-align: right !important;
+      background: none !important;
+      // overflow: hidden !important;
```
Comment (humphd) [inline public/resources/remix/style.less]: Remove this vs. commenting out

**Comment by humphd** [comment]
A few nits, r=me with those fixed.

**Comment by flukeout** [comment]
@humphd Okay, made the changes you requested - let's get it up on staging and see if we're still seeing problems on mobile.

**Pull request merged**